apps/dashboard/src/utils/schema.ts

```diff
@@ -153,10 +154,14 @@ export const buildDefaultValues = (uiSchema: UiSchema): object => {
     }
 
     const { placeholder: defaultValue } = property;
-    if (defaultValue === null || typeof defaultValue === 'undefined') {
+    if (typeof defaultValue === 'undefined') {
       return acc;
     }
 
+    if (defaultValue === null) {
+      return { ...acc, [key]: defaultValue };
+    }
```

Comment:
when `null` is sent as the default value, just set it on the form (it supports null but not undefined)